File: 프로그래머스/unrated/181908. 접미사인지 확인하기/접미사인지 확인하기.cs
using System;

public class Solution {
    public int solution(string my_string, string is_suffix) {
        int index = my_string.LastIndexOf(is_suffix);
        if(index == -1) return 0;
        return index == my_string.Length - is_suffix.Length ? 1 : 0;
    }
}

File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string my_string, string is_suffix) {
    
    if (my_string.size() < is_suffix.size())
        return 0;
    
    int length = is_suffix.size();
    string substr = my_string.substr(my_string.size() - length, length);
    return substr == is_suffix;
}